modules/profile/views_handler_field_profile_list.inc

```diff
@@ -11,7 +11,7 @@ class views_handler_field_profile_list extends views_handler_field_prerender_lis
     foreach ($values as $value) {
       $field = $value->{$this->field_alias};
       $this->items[$field] = array();
-      foreach (split("[,\n\r]", $field) as $item) {
+      foreach (preg_split("/[,\n\r]/", $field) as $item) {
```

Comment:
D6LTS 6.45 solved this by introducing a shim for split(), on the bottom of bootstrap.inc